bindgen/bindgen.bzl

```diff
@@ -0,0 +1,167 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
```

Comment:
Added file should have the year of the addition:
```suggestion
# Copyright 2019 The Bazel Authors. All rights reserved.
```

Reply:
Can do. 

Is "The Bazel Authors" also correct for this project?